servers/src/main/java/tachyon/master/TachyonMaster.java

```diff
@@ -102,23 +102,24 @@ public TachyonMaster(TachyonConf tachyonConf) {
     TachyonConf.assertValidPort(port, mTachyonConf);
     TachyonConf.assertValidPort(webPort, mTachyonConf);
 
-    String hostnameExternal = mTachyonConf.get(Constants.MASTER_HOSTNAME, "localhost");
-    // The master listens to the MASTER_HOSTNAME_LISTENING configuration option.
-    // MASTER_HOSTNAME_LISTENING defaults to MASTER_HOSTNAME if unspecified. If set to
-    // MASTER_HOSTNAME_LISTENING_WILDCARD, server listens to all addresses.
-    String hostnameListening =
-        mTachyonConf.get(Constants.MASTER_HOSTNAME_LISTENING, hostnameExternal);
-
-    InetSocketAddress address = new InetSocketAddress(hostnameExternal, port);
-    InetSocketAddress addressListening;
-    if (hostnameListening.equals(Constants.MASTER_HOSTNAME_LISTENING_WILDCARD)) {
-      addressListening = new InetSocketAddress(port);
-    } else {
-      addressListening = new InetSocketAddress(hostnameListening, port);
-    }
+    // String hostnameExternal = mTachyonConf.get(Constants.MASTER_HOSTNAME, "localhost");
+    // // The master listens to the MASTER_HOSTNAME_LISTENING configuration option.
+    // // MASTER_HOSTNAME_LISTENING defaults to MASTER_HOSTNAME if unspecified. If set to
+    // // MASTER_HOSTNAME_LISTENING_WILDCARD, server listens to all addresses.
+    // String hostnameListening =
+    // mTachyonConf.get(Constants.MASTER_HOSTNAME_LISTENING, hostnameExternal);
+    //
+    // InetSocketAddress address = new InetSocketAddress(hostnameExternal, port);
+    // InetSocketAddress addressListening;
+    // if (hostnameListening.equals(Constants.MASTER_HOSTNAME_LISTENING_WILDCARD)) {
+    // addressListening = new InetSocketAddress(port);
+    // } else {
+    // addressListening = new InetSocketAddress(hostnameListening, port);
+    // }
+    InetSocketAddress masterBindAddress = NetworkAddressUtils.getMasterBindAddress(mTachyonConf);
```

Comment:
You commented out all the code that handles the master hostname listening and wildcard logic. This is because after the change it will be handled transparently by setting bind address to 0.0.0.0?